front/components/DataSourceResourceSelectorTree.tsx

```diff
@@ -27,7 +28,7 @@ export default function DataSourceResourceSelectorTree({
   viewType = "documents",
 }: {
   owner: WorkspaceType;
-  dataSource: DataSourceType;
+  dataSource: DataSourceType | DataSourceViewType;
```

Comment:
If we can do:
```suggestion
  dataSourceOrView: DataSourceType | DataSourceViewType;
```

Reply:
Will do in a dedicated PR because this touches all lot of files and I much rather not have this PR bigger than it already is. 👍🏻

Reply:
@flvndvd  https://github.com/dust-tt/dust/pull/6866